Добавлены цвета, темы, строковые ресурсы

AnShok opened the pull request.

AnShok committed:
Добавлены цвета, темы, строковые ресурсы

AnShok committed:
Исправлена ошибка в строковых ресурсах

Logomann commented on app/src/main/res/values-night/themes.xml:
```diff
@@ -1,7 +1,29 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
     <style name="Base.Theme.PracticumAndroidDiploma" parent="Theme.Material3.DayNight.NoActionBar">
```
Тема по-умолчанию (Theme.Material3) изначально идет с форка яндекса, а в задание указано: "Светлая и тёмная темы приложения наследуются от Theme.MaterialComponents.DayNight.NoActionBar", критично ли использовать новую версию вместо старой?

ElchinGasymov replied:
Давайте заменим на parent="Theme.MaterialComponents.DayNight.NoActionBar"

Logomann commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,3 +1,61 @@
+    <string name="name3">Молоданов Виталий Олегович</string>
+    <string name="name4">Шокин Андрей Александрович</string>
+    <string name="name5">Рябицкий Андрей Игоревич</string>
+
```
Кажется не хватает еще добавить текст <string name="vacancy_list" >Найдено %d вакансий</string>

Logomann commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,3 +1,61 @@
+    <string name="server_error">Ошибка сервера</string>
+    <string name="favorite_list_empty">Список пуст</string>
+    <string name="there_no_such_vacancies">Таких вакансий нет</string>
+    <string name="no_vacancy_list">Не удалось получить\n список вакансий</string>
```
Дублируется текст выше, в 9 строке

ElchinGasymov commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,3 +1,61 @@
     <string name="app_name">Practicum-Android-Diploma</string>
-</resources>
\ No newline at end of file
+    <string name="main_screen_title">Главная</string>
```
Зачем перенос строки?

ElchinGasymov commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,3 +1,61 @@
+    <string name="contact_person">Контактное лицо</string>
+    <string name="email_title">E-mail</string>
+    <string name="phone_title">Телефон</string>
+
```
а тут целых 2

AnShok committed:
Убраны дубли в strings.xml
Изменен родительский стиль темы приложения с Theme.Material3.DayNight.NoActionBar на Theme.MaterialComponents.DayNight.NoActionBar
Добавлен строковый ресурс для отображения количества найденных вакансий.

AnShok committed:
Убраны дубли в strings.xml
Изменен родительский стиль темы приложения с Theme.Material3.DayNight.NoActionBar на Theme.MaterialComponents.DayNight.NoActionBar
Добавлен строковый ресурс для отображения количества найденных вакансий.
